packages/playground/src/types.ts

```diff
@@ -255,23 +250,23 @@ export interface SensitivityMatcher {
   fieldMappings?: DocumentsFieldMappings;
   sensitiveSecurityCategory?: number;
   restrictToSources?: string[];
+  filterPasswords?: boolean;
 }
 
 export interface DocumentsFieldMappings {
-  title?: string;
-  author?: string;
-  mimeType?: string;
-  type?: string;
+  title?: string | string[];
+  author?: string | string[];
+  mimeType?: string | string[];
+  type?: string | string[];
```

Comment:
The API will always return `string[]`, but support both `string` and `string[]` in requests. The API supports both on request to be backwards compatible, but we want to phase out the `string` option. So, can they be only `string[]` here?